### ISSUE TITLE
Could you review Brave?

### ISSUE THREAD

**Issue by Villagerjj** [PlayTooMuchGames/awesome-browsers#1]
I use brave all the time, it is based on chromium but with adblocker and VPN capabilities. also is detected as chrome on every website.

**Comment by PlayTooMuchGames** [comment]
Yup, I was going to do Brave and many other browsers later on. As you can probably tell, this list is still very much WIP. stay tuned as I update it!
